Swift/Replicator.swift

```diff
@@ -98,16 +98,29 @@ public final class Replicator {
         _impl.start()
     }
     
+    /// Starts the replicator with an option to reset the local checkpoint of the replicator. When the local checkpoint
+    /// is reset, the replicator will sync all changes since the beginning of time from the remote database.
+    /// This method returns immediately; the replicator runs asynchronously and will report its progress throuh
+    /// the replicator change notification.
+    ///
+    /// - Parameters:
+    ///   - reset: Reset the local checkpoint before starting the replicator.
```

Comment:
is this "Reset the local checkpoint " or "Resets the local checkpoint " ?

Reply:
The later is better.